include/kodlab_mjbots_sdk/jointBase.h

```diff
@@ -0,0 +1,84 @@
+
+//
+// Created by kodlab on 02/17/22.
+// J. Diego Caporale
+//
+/**
+ * @file jointBase.h
+ * @author Kodlab - J. Diego Caporale (jdcap@seas.upenn.edu)
+ * @brief Abstract class for joints. Agnostic to servo/motor used.
+ * @version 0.1
+ * @date 2022-02-17
+ * 
+ * @copyright Copyright (c) 2022
+ * 
+ */
+#include <limits>
+
+
+
+// struct joint_options{
+    
+// }
+
+
+/**
+ * @brief 
+ * Abstract Base class for any joint used in the robot class.
+ */
+class JointBase {
+    protected:
+        float gear_ratio_ = 1.0;
+        float zero_offset_ = 0;
+        int  direction_ = 1; 
+        float max_torque_ = std::numeric_limits<float>::infinity();
+        float pos_limit_min_ = -std::numeric_limits<float>::infinity();
```

Comment:
Will need to describe what these actually mean. What happens if you exceed that limit?

Reply:
yes agreed wanted to run by you what the plan would be for these. also the defaults are describe in too many places maybe